src/lib/utils/auto-prefixer.ts

```diff
@@ -10,24 +10,21 @@
 export function applyCssPrefixes(target) {
   for (let key in target) {
 
-    let value = target[key];
+    let value = target[key] || "";
+    let boxValue = toBoxValue(value);
+    let boxDirection = toBoxDirection(value);
+    let boxOrient = toBoxOrient(value);
 
     switch (key) {
       case 'display':
         if (value === 'flex') {
           target['display'] = [
             '-webkit-box',
-            '-moz-box',
-            '-ms-flexbox',
-            '-webkit-flex',
```

Comment:
@jelbourn - I am very confused here. [AutoPrefixer](https://autoprefixer.github.io/) uses `-webkit-box` and does **NOT** use `-webkit-flex`. Your comment ^ and some research proves that source wrong. 

Here is what I found:

```css
{
 display: -webkit-box;  /* OLD - iOS 6-, Safari 3.1-6, BB7 */
  display: -ms-flexbox;  /* TWEENER - IE 10 */
  display: -webkit-flex; /* NEW - Safari 6.1+. iOS 7.1+, BB10 */
  display: flex;         /* NEW, Spec - Firefox, Chrome, Opera */
}
```

I will restore the `-webkit-flex` settings.  Thx for the oversight.